Justfile

```
@@ -6,6 +6,26 @@ release:
     cargo build --release
     @just copy-binary
 
+# Build Windows executable
+release-windows:
+    #!/usr/bin/env sh
+    if [ "$(uname)" = "Darwin" ] || [ "$(uname)" = "Linux" ]; then
+        echo "Building Windows executable using Docker..."
+        docker run --rm -v "$(pwd)":/usr/src/myapp -w /usr/src/myapp \
+            rust:latest \
+            sh -c "rustup target add x86_64-pc-windows-gnu && \
+                apt-get update && \
+                apt-get install -y mingw-w64 && \
+                cargo build --release --target x86_64-pc-windows-gnu && \
+                cp /usr/lib/gcc/x86_64-w64-mingw32/*/libstdc++-6.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/ && \
+                cp /usr/lib/gcc/x86_64-w64-mingw32/*/libgcc_s_seh-1.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/ && \
+                cp /usr/x86_64-w64-mingw32/lib/libwinpthread-1.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/"
+    else
+        echo "Building Windows executable natively..."
+        powershell.exe -Command "docker run --rm -v ${PWD}:/usr/src/myapp -w /usr/src/myapp rust:latest sh -c 'rustup target add x86_64-pc-windows-gnu && apt-get update && apt-get install -y mingw-w64 && cargo build --release --target x86_64-pc-windows-gnu && cp /usr/lib/gcc/x86_64-w64-mingw32/*/libstdc++-6.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/ && cp /usr/lib/gcc/x86_64-w64-mingw32/*/libgcc_s_seh-1.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/ && cp /usr/x86_64-w64-mingw32/lib/libwinpthread-1.dll /usr/src/myapp/target/x86_64-pc-windows-gnu/release/'"
+    fi
+    @echo "Windows executable and required DLLs created at ./target/x86_64-pc-windows-gnu/release/"
```

```suggestion
    echo "Windows executable and required DLLs created at ./target/x86_64-pc-windows-gnu/release/"
```
It looks like we shouldn't have this `@`: `release-windows: line 27: @echo: command not found`